src/includes/performance/traces-sampler-as-sampler/dotnet.aspnetcore.mdx

```diff
@@ -0,0 +1,38 @@
+```csharp
+using Sentry;
```

Comment:
We need the using to access some extension method? Otherwise there's no need here since we're just doing "options."

Reply:
Merging for now as this fixes a big issue. Please remove the using if needed in a follow up pr